installers/binstall/src/system/env.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Since we are putting this file on people's computer, can we add more comments about what it is doing and what users should do if they come across it?

Reply:
done!